lib/active_admin/application.rb

```diff
@@ -40,6 +40,12 @@ def initialize
     # Set a favicon
     inheritable_setting :favicon, false
 
+    # Additional meta tags to place in head of logged in pages.
+    inheritable_setting :meta_tags, {}
+
+    # Additional meta tags to place in head of logged out pages.
+    inheritable_setting :meta_tags_for_logged_out_pages, { robots: 'noindex, nofollow' }
```

Comment:
Line is too long. [88/80]<br>Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.